docker-compose.yml

```diff
@@ -100,14 +103,14 @@ services:
     environment:
       << : *default-environment
 
-  solr:
-    image: amazeeio/solr:6.6-drupal
-    labels:
-      lagoon.type: solr
-    ports:
-      - "8983" # exposes the port 8983 with a random local port, find it with `docker-compose port solr 8983`
-    environment:
-      << : *default-environment
+#  solr:
+#    image: amazeeio/solr:6.6-drupal
+#    labels:
+#      lagoon.type: solr
+#    ports:
+#      - "8983" # exposes the port 8983 with a random local port, find it with `docker-compose port solr 8983`
+#    environment:
+#      << : *default-environment
```

Comment:
Why is this commented out? Seems unrelated to Redis

Reply:
It was a leftover of my local setup. fixed